pkg/randchar/generator.go

```diff
@@ -244,6 +244,30 @@ func NewCharset(characters string) Charset {
 	}
 }
 
+// GetCharset returns the charset with the specified name.
```

Comment:
godoc is still named `GetCharset`, while the function has been renamed to `CharsetByName`.

Reply:
Fixed it!